plopfile.js

```diff
@@ -0,0 +1,36 @@
+module.exports = (plop) => {
```

Comment:
lol are we using js or ts? Should we keep the language consistent?

Reply:
Whoops! Good catch.